### DISCUSSION TITLE
Configuring users in STF local

### DISCUSSION THREAD

**Discussion by nabuah** [DeviceFarmer/stf#810] [Q&A]
Hello,

I am running STF local, and I was wondering if I can configure additional users with username and password in this mode.

What I have is a simple setup with 4 devices, that I need to be accessible remotely by my coworkers in the office.
Because the setup is very minimalistic, I figured that I don't need a proper production deployment and I can use "stf local" mode.

However, I do prefer to configure additional users with password for each. 

Is it something that is possible? I don't need ldap authentification or something complicated, locally created usernames and passwords will be enough.

Thanks in advance.

**Comment by denis99999** [comment]
@nabuah , you can use mock authentication and create a `standard` or `bookable` group (as admin user) for your 4 devices and your colleagues only so that other users outside of this group cannot see or take control of your devices

**Reply by nabuah** [reply]
Got it, thanks for the suggestion!
